src/components/switch/readme.md

```diff
@@ -0,0 +1,75 @@
+# gx-switch
+
+An input element for switch/toggle options.
+
+## Children
```

Comment:
I'd remove this section. See the review comment for the readme.md file of gx-group, please.